todo_app/app.py

```diff
@@ -1,78 +1,88 @@
 from flask import Flask, render_template, redirect, request
-from todo_app.flask_config import Config
 import requests
 import os
 from todo_app.data.todo_item import TodoItem
+from todo_app.data.view_model import ViewModel
 
-app = Flask(__name__)
 
-@app.route('/')
-def index():
-    url = f"https://api.trello.com/1/boards/{os.environ.get('TRELLO_BOARD_ID')}/cards"
 
-    query = {
-        'key': os.environ.get("TRELLO_KEY"),
-        'token': os.environ.get("TRELLO_TOKEN")
-    }
+def create_app():
+    app = Flask(__name__)
+    #app.config.from_object('app_config.Config')
```

Comment:
You're right that you don't actually need app_config.Config anymore (it was used for SECRET_KEY for encrypting cookies).

But you should either fix it or delete it. 

To fix it, `import app_config`, change the Config class to set self.SECRET_KEY inside an `__init__(self)` function and change this to 
```suggestion
    app.config.from_object(app_config.Config())
```